lib/Sema/TypeCheckDeclPrimary.cpp

```diff
@@ -1960,9 +1960,23 @@ class DeclChecker : public DeclVisitor<DeclChecker> {
     // diagnostics.
     (void)ID->getDecls();
 
+    auto target = ID->getModule();
+    if (target->isNonUserModule() && // target module is in distributed SDK
+        getASTContext().LangOpts.PackageName == target->getPackageName().str()) {
```

Comment:
Should we check that the package name is not empty to avoid this warning between two modules without a package name?

Reply:
good point, will update